policies/branch-policy.md

```diff
@@ -0,0 +1,100 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+## The master branch
+
+- Any type (bug fix, feature, refactoring…) of pull requests is allowed.
+
+- The development of the next minor or major release happens there.
+  Which kind of release it is is defined by OMC at some point after the
+  master branch is opened after the previous release was branched.
+  Implicitly it means that any API/ABI breaking changes are not allowed
+  until OMC defines the intended kind (minor/major) of the next release.
```

Comment:
I think that having extended periods of time where API/ABI-breaking changes are not allowed introduces a large risk of slowing down development.  If large projects/refactorings might be stalled for years (i.e., if next-minor is not approved), there is reduced incentive to start on them and incur the ongoing maintenance burden to keep them up to date until they can be reviewed and merged.

We already have numerous data points of submissions from the community of large features that have been awaiting review for literally years, with no sign of such review arriving anytime soon, either.  Please consider the effects of this lack of motion on the broader community.